android-components/components/ui/icons/src/main/res/drawable/mozac_ic_chevron_down_20.xml

```diff
@@ -0,0 +1,12 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="20dp"
+    android:height="20dp"
+    android:viewportWidth="20"
+    android:viewportHeight="20">
+    <path
+        android:fillColor="@color/mozac_ui_icons_fill"
+        android:pathData="M10,12.19 L5.06,7.25 4,8.31l5.47,5.47a0.75,0.75 0,0 0,1.06 0L16,8.31l-1.061,-1.06L10,12.19z" />
```

Comment:
Curious about the `android:fillType="evenOdd"` removal here from https://github.com/FirefoxUX/acorn-icons/blob/main/icons/mobile/20/xml/ic_chevron_down_20.xml? Was this intentional? Is this something we should generally remove elsewhere?

Reply:
It was an android studio warning that android doesn't support this < 24 and it works fine without so I removed it. Although android supports this for a while with support vector drawables and we have references of this in code. None of the icons in `ui-icons` seem to use it. I'm not sure if this is required for this vector as there are no holes to fill. [reference](https://medium.com/thoughts-overflow/the-mystery-of-the-disappearing-holes-a-gripping-tale-of-using-svg-in-android-442f6035a452). I don't know know much about this property at this moment to say much. wdyt?

Also, found this while I was removing this --> https://github.com/FirefoxUX/photon-icons/issues/28

Reply:
Thanks, I was able to reproduce the error. The only unfortunate part of this dependency chain is that I wished we had the changes upstream in https://github.com/FirefoxUX/acorn-icons/blob/main/icons/mobile/20/xml/ic_chevron_down_20.xml as well. I will make a mental note and make these upstream changes when we are ready to copy all the XML sources from the acorn-icons repo and able to identify all the upstream changes that are needed. No further action necessary here.